charts/grafana/FLUX.md

```diff
@@ -1,12 +1,24 @@
 # Flux language setup
-### Manual Setup
-* Flux is still in beta, so this information may be out of date
-* Follow instructions laid out by [Flux Datasource for Grafana]
-(https://grafana.com/plugins/grafana-influxdb-flux-datasource)
+## Manual Setup
+- Flux is still in beta, so this information may be out of date
+- Enable Flux by setting the flux-enabled option to true under the http section of your influxdb.conf
```

Comment:
`flux-enabled` , `true` , `[http]` , influxdb.conf should be prefaced by the name of the enclosing configmap in influxdb/templates , [`config.yaml`](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/influxdb/templates/config.yaml). Just make it look nice, make it easy to tell what's code, and give references to what you're referencing.

Reply:
`[http]` not `http`
That is a section in the config file and the brackets help denote that.